container/openejb-core/src/test/java/org/apache/openejb/assembler/classic/ImportSqlScriptTest.java

```diff
@@ -33,11 +37,18 @@
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.Id;
 import jakarta.persistence.PersistenceContext;
-import java.util.Properties;
 
-import static org.junit.Assert.assertEquals;
 
+/**
+ * This class should no longer be used to test the ImportSql class.
```

Comment:
This class comment can be removed. It doesn't serve any purpose since `@Deprecation` is enough here. Please also restore previous import order.